constants/providersContent.ts

```diff
@@ -9,13 +12,24 @@ type TableOfContentEntry = {
   isSecondary?: boolean
 }
 
-type ProvidersSubSection = {
+type ProviderObject = {
   title: string
-  content: {
-    image: string
-    description: string
-    cta: NavLink
+  image: string | null
+  description: string
+  installation: string
+  websiteCta: {
+    label: string | null
+    url: string | null
+  }
```

Comment:
```suggestion
  websiteCta?: GeneralLink
```

Same for `docsCta` and `sourceCta`

Also, it would be nice to have info for segment on these links

Reply:
True

updated in https://github.com/Qiskit/qiskit.org/pull/3000/commits/88ac621a65a1abc7d1e8637babbe6b846672e819